src/native/mac/qgsmacnative.mm

```diff
@@ -121,5 +121,9 @@ - ( BOOL )userNotificationCenter:( NSUserNotificationCenter * )center shouldPres
 
 bool QgsMacNative::hasDarkTheme()
 {
-  return ( NSApp.effectiveAppearance.name != NSAppearanceNameAqua );
+  #if __MAC_OS_X_VERSION_MIN_REQUIRED >= 101400
```

Comment:
I am not sure this is safe
https://stackoverflow.com/questions/44318151/why-are-mac-os-x-version-min-required-and-mac-os-x-version-max-allowed-no-longer
It might be that min required version (I think provided by Qt) is not automatically set to the one of the OS?
We could use something like `QSysInfo::productVersion().toDouble() >= 10.14`
http://doc.qt.io/qt-5/qsysinfo.html#productVersion

Reply:
oops, no sorry it won't build.